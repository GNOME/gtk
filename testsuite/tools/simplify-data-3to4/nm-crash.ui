<interface domain="nm-applet">
  <object class="GtkAssistant">
    <child>
      <object class="GtkBox" id="confirm_page">
      </object>
      <packing>
        <property name="page_type">confirm</property>
        <property name="complete">True</property>
      </packing>
    </child>
  </object>
</interface>
