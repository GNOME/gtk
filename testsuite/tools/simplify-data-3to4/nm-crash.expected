<?xml version="1.0" encoding="UTF-8"?>
<interface domain="nm-applet">
  <object class="GtkAssistant">
    <child>
      <object class="GtkAssistantPage">
        <property name="page_type">confirm</property>
        <property name="complete">True</property>
        <property name="child">
          <object class="GtkBox" id="confirm_page">
            <property name="visible">0</property>
          </object>
        </property>
      </object>
    </child>
  </object>
</interface>
